docs/content/items.md

```diff
@@ -271,21 +271,31 @@ Batch actions are as follows:
 - Delete selected: delete only the items that are selected on the page
 - Delete all: delete all the items returned by a search (default is all items).
 
-Choose one of these options and then click "Go".
+The number of results per page is set in the [installation's global settings](../admin/settings.md#general).
+
+You can first use the "🔍 Advanced search" link to narrow down the items to a subset for editing. For example, you can use [Advanced search](../search.md#item-advanced-search) to limit to items that are not in any item sets, or items that are owned by a specific user. In the image below, the search is set to capture any item without a "Date created" value. 
```

Comment:
Line 276: Unicode magnifying glass icon, yes or no? I kinda like it but I haven't totally looked into rendering issues yet.